### PR TITLE
Use newly factored-out @wikmedia/url-get library

### PR DESCRIPTION
**Pull request by ottomata**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,15 +50,16 @@
     "preq": "^0.5.6",
     "service-runner": "^2.7.7",
     "swagger-router": "^0.7.1",
-    "swagger-ui": "git+https://github.com/wikimedia/swagger-ui#master"
+    "swagger-ui": "git+https://github.com/wikimedia/swagger-ui#master",
```
Comment (Pchelolo) [inline package.json:53]: unrelated: you should probably update to a newer service-template-node, we've retired this fork long time ago.

**Review by Pchelolo — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "eventgate",
-  "version": "1.5.1",
+  "version": "1.5.2",
```
Comment (Pchelolo) [inline package.json]: Technically this is a breaking change since you don't reexport the util functions.. so maybe at least 1.6.0?

Comment (ottomata) [reply package.json]: Done

File: package.json (hunk 1 above)
Comment (ottomata) [reply package.json:53]: Yeah...

**Comment by Pchelolo** [comment]
LGTM

**Pull request merged**